newsletter.md

```diff
@@ -0,0 +1,14 @@
+
+During the past few months, the team has been working hard to make new features available in the OpenPaaS MVP release.
+
+
+The team implemented JMAP sharing, allowing one user to share a folder with another user of the same domain. The sharee gets the right, if allowed, to read, mark, delete and moves emails. This is especially useful to delegate the management of your emails to someone else. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting JMAP sharing feature in INBOX.
+
```

Comment:
"This is especially useful to delegate the management of your emails to someone else": I hope this sentence won't confuse the reader with the delegation that a user could have on the whole inbox module (which we know does not exist as well as delegating an OP account does not exist)

Reply:
s/your emails/a specific mailbox/